doozer/doozerlib/cli/images_health.py

```diff
@@ -85,9 +132,10 @@ def url_text(url, text):
         if latest_success_idx == -1:
             # No success record was found
             msg += f'Failing for at least the last {len(records)} attempts / {oldest_attempt_bi_dt}'
+            msg += f'. See more details: {art_dash_link}'
```

Comment:
I think this line is not necessary

Reply:
I think we should provide art-dash link on build failure, no matter if the latest build is failing or not.